pkgs/development/tools/wails/default.nix

```diff
@@ -0,0 +1,91 @@
+{ lib
+, stdenv
+, buildGoModule
+, fetchFromGitHub
+, pkg-config
+, makeWrapper
+, go
+, gcc
+, gtk3
+, webkitgtk
+, nodejs
+, upx
+, zlib
+}:
+
+buildGoModule rec {
+  pname = "wails";
+  version = "2.0.0-beta.33";
+
+  src = fetchFromGitHub
+    {
+      owner = "wailsapp";
+      repo = pname;
+      rev = "v${version}";
+      sha256 = "efxOL/FllToum0P4JyAJt0fbrznTFYh7czTWpZu3uk0=";
+    } + "/v2";
+
+  vendorSha256 = "qPMVsvud2L7hpXUOfYYMiO32JXff8ZZC34EsxFoSJ0g=";
```

Comment:
Isn't this a base64 encoded sri hash? If so the sha256- should be prefixed again.

Reply:
Yeah, usually only `sha256:` is stripped. Later `sha256` could be renamed to `hash` and everything can continue to work.